cfn-guard-lambda/README.md

```diff
@@ -18,6 +18,7 @@ The primary interface for building and deploying the tool is the [Makefile](Make
 * [Rust](https://rustup.rs/) (See the installation instructions in the [top-level README](../README.md#install-rust))
 * If building on a Mac, you'll need [Homebrew](https://brew.sh/).  
 * If building on Ubuntu, you'll need to run `sudo apt-get update; sudo apt install build-essential` if you haven't already
+* If building on CentOS/RHEL you'll need to install `jq`, `gcc` and the `musl-devel` package (see https://copr.fedorainfracloud.org/coprs/ngompa/musl-libc/ for instructions to add the musl-libc repository to your yum config)
```

Comment:
Since the `make pre-reqs` command does the install here, could we just make this:

"you'll need to add the musl-libc repository to your yum config: <url>"

?